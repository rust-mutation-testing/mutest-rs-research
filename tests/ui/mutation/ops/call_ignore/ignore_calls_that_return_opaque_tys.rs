//@ print-mutants
//@ build
//@ stdout
//@ stderr: empty
//@ mutation-operators: call_delete, call_value_default_shadow

fn ascii_letters(uppercase: bool) -> impl Iterator<Item = char> {
    match uppercase {
        true => 'A'..'Z',
        false => 'a'..'z',
    }
}

fn f() {
    let _ = ascii_letters(true);
}

#[test]
fn test() {
    f();
}
