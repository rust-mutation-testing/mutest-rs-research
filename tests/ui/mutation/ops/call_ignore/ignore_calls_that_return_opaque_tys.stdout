0 mutants; 0 mutations; 0 safe; 0 unsafe (0 tainted); 0 batched; 0 unbatched
